### ISSUE TITLE
store sbom result on OCI registry along with an image

### ISSUE THREAD

**Issue by developer-guy** [ckotzbauer/sbom-operator#22]
Hello @ckotzbauer, in cosign there is a support for attaching SBOM files to an OCI registry along with an image. So, maybe we can support that too as an alternative way of storing SBOM files instead of just storing them in git. WDYT?

**Comment by ckotzbauer** [comment]
Hi @developer-guy, that sounds really interesting. But that only would make sense for "own" images, where you have write-permissions to store those SBOMs along with the image, right?

**Comment by developer-guy** [comment]
exactly! great catch ❤️

**Comment by ribbybibby** [comment]
> But that only would make sense for "own" images, where you have write-permissions to store those SBOMs along with the image, right?

Not necessarily. Cosign supports the option to store the SBOM in a different repository to the target image.

**Comment by ckotzbauer** [comment]
Yes, correct. But that would store all sboms in one repo without its images. Could be useful, however.

**Comment by ribbybibby** [comment]
> Could be useful, however.

I think it would be, yep. 

For the use case I'm thinking about, I'd like to generate SBOMs for all the images I'm running in Kube and store them in a single repository (i.e `example.com/myorg/sboms`).

Then I can download any SBOM with cosign thusly:
```
COSIGN_REPOSITORY=example.com/myorg/sboms cosign download sbom prom/prometheus:v2.30.0
```

**Comment by ckotzbauer** [comment]
Yes, that would be a great feature for this project. Any PRs are welcome for this topic.
